_about/notes/0.5.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Istio 0.5
+
+order: 96
+
+layout: about
+type: markdown
+---
+{% include home.html %}
+
+In addition to the usual pile of bug fixes and perf improvements, this release includes the new or 
+updated features detailed below.
+
+## Networking
+
+- **Alpha2 Traffic Rules**. Add support for an updated model of routing and destination rules. TODO: Doc link?
```

Comment:
is this ready to be announced?  @rshriram @ijsnellf @GregHanson

Reply:
Its partly ready. I would qualify that unlike alpha1, this is hot off the oven. Likely to be a bumpy ride

Reply:
Should we remove it then? Or move it lower in the list? Different wording?

Reply:
Lower on the list, probably. We have support for ~70% of the new APIs, but it's likely we'll be spending quite a bit of time debugging/improving (they truly are "alpha" atm).

Reply:
Yeah. lower in the list.

Reply:
OK, I moved it down the list and reworded to add some caveats.